Update docs (#73)
layerai/dbt-layer: examples/order_review_prediction/README.md

> # Predicting Order Review Scores in dbt DAG

Reviewer: ```suggestion
# Predicting Order Review Scores in the dbt DAG
```

> # Predicting Order Review Scores in dbt DAG

Reviewer 2: ```suggestion
# Predicting Order Review Scores from within the dbt DAG
```

> In this e-commerce example walk through, we will train a machine learning model to predict order review scores in the middle of dbt DAG with the opensource [Layer dbt Adapter](https://github.com/layerai/dbt-adapters). The Layer dbt Adapter lets you easily enhance your dbt pipelines with machine learning (ML) workloads.

Reviewer 2: ```suggestion
In this e-commerce example walk-through, we train a machine learning model to predict order review scores integrated within a dbt DAG, using  the open source [Layer dbt Adapter](https://github.com/layerai/dbt-adapters). The Layer dbt Adapter lets you easily enhance your dbt pipelines with machine learning (ML) workloads.
```

> We will use `layer.automl()` functionality to train an AutoML model based on the Brazilian e-commerce company, [OLIST’s datasets](https://www.kaggle.com/datasets/olistbr/brazilian-ecommerce).  

Reviewer 2: ```suggestion
We use `layer.automl()` functionality to train an AutoML model based on the Brazilian e-commerce company, [OLIST’s datasets](https://www.kaggle.com/datasets/olistbr/brazilian-ecommerce).  
```

> This is all the SQL you need to train your ML model within the dbt DAG. You just pass Layer the model type, input features and the target column we want to predict:

Reviewer 2: (changing person) If we want to use `you` here, we should change all previous steps from `we` to `you` as well.

```

Once you run this SQL:

> 1. Layer will try different machine learning algorithms with many parameters to find the best one for your data

Reviewer 2: ```suggestion
1. Layer tries different machine learning algorithms with many parameters to find the best one for your data
```

> 2. Once the training done, Layer will register the model to your Layer account. [Click here](https://app.layer.ai/layer/order_review_prediction/models/review_score_predictor) for a sample AutoML model registered on Layer. 

Reviewer 2: ```suggestion
2. Once the training done, Layer registers the model to your Layer account. [Click here](https://app.layer.ai/layer/order_review_prediction/models/review_score_predictor) for a sample AutoML model registered on Layer. 
```

> 3. Then, you will be able to fetch the model and use it for predictions inside dbt DAG with `layer.predict()`. [Click here](./models/predictions.sql) to see a prediction example in this project.

Reviewer 2: ```suggestion
3. Then, you can fetch the model and use it for predictions from within dbt DAG with `layer.predict()`. [Click here](./models/predictions.sql) to see a prediction example in this project.
```

```

> 4. Now, we are ready to run our dbt DAG. To get the code, clone this repo, and go to the `order_review_prediction` example folder:

Reviewer 2: Change _you/we_ corresponding to the choices made above.
